tools/gbagfx/jasc_pal.c

```diff
@@ -9,58 +9,73 @@
 // Read/write Paint Shop Pro palette files.
 
 // Format of a Paint Shop Pro palette file, line by line:
-// "JASC-PAL\r\n" (signature)
-// "0100\r\n" (version; seems to always be "0100")
-// "<NUMBER_OF_COLORS>\r\n" (number of colors in decimal)
+// "JASC-PAL" (signature)
+// "0100" (version; seems to always be "0100")
+// "<NUMBER_OF_COLORS>" (number of colors in decimal)
 //
 // <NUMBER_OF_COLORS> times:
-// "<RED> <GREEN> <BLUE>\r\n" (color entry)
+// "<RED> <GREEN> <BLUE> <ALPHA (optional)>" (color entry)
```

Comment:
It might be worth noting here that `ALPHA` is discarded, in case someone is expecting to use it.